resource-managers/kubernetes/core/pom.xml

```diff
@@ -89,6 +88,12 @@
     </dependency>
     <!-- End of shaded deps. -->
 
+    <dependency>
```

Comment:
Why does this become necessary, out of curiosity? looks like a downgrade. I'm just wondering if it affects anything else that depends on okhttp.

Reply:
I agree with @srowen . This looks suspicious.

Reply:
this was my mistake -reverted.